### PR TITLE
Minor formatting changes

### PR DESCRIPTION
**Pull request by skarthikeya**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ propose changes to this document in a pull request.
 but before doing that please read the notes below and include as many details as
 possible with your report. If you can, please include:
   * The version of Electron you are using
-  * The operating system you are using
-  * If applicable, what you were doing when the issue arose and what you
-  expected to happen
+  * The specifications of the operating system you are using
```
Comment (kevinsawicki) [inline CONTRIBUTING.md]: I think the previous version of this sentence was clearer and this addition feels unneeded. Can you clarify a bit on why it was updated?

Comment (skarthikeya) [inline CONTRIBUTING.md]: Asking for Operating system would not be sufficient I felt. We need to know the specifications of it to understand the issue more concisely. I might be wrong, but that is what I was thinking.

Comment (MarshallOfSound) [inline CONTRIBUTING.md]: @skarthikeya All we normally need is which OS so we can determine windows/macos/linux so if we need to reproduce the issue we know which OS to try.  The specs of the machine don't affect it 99.99% of the time

**Comment by kevinsawicki** [comment]
Closing this out for now we can reopen when it gets updated, thanks.

**Pull request closed without merging**